src/components/pages/public/MovieDetailPage.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div>
+    Single movie of id:
+    {{ $route.params.movieId }}
```

Comment:
We could try to use the props to pass the movieId, which (at least in my humble opinion 😅) would make a bit more cleaner and easier to use.

You can set them up in route definition and then here define the `props` option in the component definition 🙌